src/web/styles.scss

```diff
@@ -283,3 +283,11 @@ h6,
   background: white;
   color: black;
 }
+
+// text-break class on current version of bootstrap (4.5.0) does not work on some tags.
+// This class can be replaced with text-break once bootstrap version is upgraded.
+// See https://github.com/twbs/bootstrap/issues/30803
+.text-break-class {
```

Comment:
The word `class` here seems useless, since this is marked as a css class with the dot in front of it. What do you think about just `text-break`?

Reply:
My original intention here was to choose a different name from `text-break` as other classes may use the current broken implementation of `text-break` class and I was a bit afraid that changing the behaviour of `text-break` might lead to regressions. 

However, after giving this some thought, maybe it might be better to override the `text-break` class with the correct implementation so that future contributors who use the `text-break` class get the expected implementation. I could also check the all current uses of `text-break` to ensure that there are no regressions.

I'll take a closer look at this and fix the issues above tonight!

Reply:
I renamed `text-break-class` to `text-break` and replaced the implementation with the current [bootstrap implementation of `text-break`](https://github.com/twbs/bootstrap/blob/88980659712653e9d9a00bf370e52f71babe9683/scss/utilities/_text.scss#L65).

I couldn't seem to replicate the issue on the course pages, the text breaks nicely for me. I've updated the implementation though, so hopefully this fixes it!

<img width="1290" alt="Screenshot 2021-07-24 at 12 53 44 AM" src="https://user-images.githubusercontent.com/60355570/126817352-1338e4ad-fd37-47ab-9fe6-e822a648869a.png">